src/components/editor/EditorArticleController.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController';
+import _ from 'lodash';
+import EditAuthorList from './EditAuthorList';
+
+const categories = [
+  "Category 1",
+  "Category 2",
+  "Category 3"
+];
+
+export default class EditorArticleController extends FalcorController {
+  constructor(props) {
+    super(props);
+    this.handleSaveChanges = this.handleSaveChanges.bind(this);
+    this.mainFormChanged = this.mainFormChanged.bind(this);
+    this.handleMainFormChange = this.handleMainFormChange.bind(this);
+    this.authorListChanged = this.authorListChanged.bind(this);
+    this.safeSetState({
+      changed: "none",
+      saving: false
+    });
+  }
+
+  static getFalcorPath(params) {
+    return [['articlesBySlug', params.slug, ['title', 'issue', 'category', 'description']], ['articlesBySlug', params.slug, 'authors', {to: 2}, ['slug', 'name']]];
+  }
+
+  handleSaveChanges(event) {
+    event.preventDefault();
+  }
+
+  mainFormChanged(event) {
+    if (event.target.name === "addAuthor") {
+      return;
+    }
+
+    let changedFlag = this.handleMainFormChange();
```

Comment:
I'm a little worried about this implementation. Basically it forces every element to check all fields everytime any letter is typed right. Maybe you should either

- Make it debounce, and run max of once per second
- Or Push checking to the responsibility of the child components, so the scope is lessened.